docs/src/pages/components/text-fields/UseInput.js

```diff
@@ -2,28 +2,40 @@ import * as React from 'react';
 import { useInput } from '@mui/base';
 import { styled } from '@mui/system';
 
+const blue = {
+  200: '#80BFFF',
+};
+
+const grey = {
+  50: '#F3F6F9',
+  100: '#EAEEF3',
+  300: '#D7DCE1',
+  400: '#BFC7CF',
+  900: '#20262D',
+};
+
 const StyledInputElement = styled('input')`
-  width: 200px;
-  font-size: 1rem;
+  width: 300px;
+  font-size: 0.875rem;
   font-family: IBM Plex Sans, sans-serif;
   font-weight: 400;
-  line-height: 1.4375em;
-  background: rgb(243, 246, 249);
-  border: 1px solid #e5e8ec;
-  border-radius: 10px;
-  padding: 6px 10px;
-  color: #20262d;
-  transition: width 300ms ease;
+  line-height: 1.5;
+  color: ${grey[900]};
+  background: ${grey[50]};
+  border: 1px solid ${grey[300]};
+  border-radius: 8px;
+  padding: 12px 12px;
+  transition: width 150ms ease;
 
   &:hover {
-    background: #eaeef3;
-    border-color: #e5e8ec;
+    background: ${grey[100]};
+    border-color: ${grey[400]};
   }
 
   &:focus {
-    outline: none;
-    width: 220px;
-    transition: width 200ms ease-out;
+    outline: 2px solid ${blue[200]};
+    width: 320px;
```

Comment:
I assume we don't need to change the width of the input once focus. In 99% of the cases, inputs don't behave like this.

```suggestion
```